genai-perf/README.md

```diff
@@ -407,9 +400,9 @@ URL of the endpoint to target for benchmarking. (default: `None`)
 ##### `--batch-size <int>`
 ##### `--batch-size-text <int>`
 
-The text batch size of the requests GenAI-Perf should send.
+The batch size of the requests GenAI-Perf should send.
 This is currently only supported with the
-[embeddings](docs/embeddings.md) and
+[embeddings](docs/embeddings.md), image_retrieval, and
```

Comment:
Similarly, image_retrieval does not support text batching, since it only uses images. You can revert to what it was before.

Reply:
done